### ISSUE TITLE
Admin: Alert analysts when domain requestor resubmits after completing "action needed"

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#2238]
### Story

As an analyst
I want to know when a user has resubmitted their request after completing an action
so that I know it's ready to review again.

### Acceptance Criteria

- [ ] consider different ways we could notify analysts when users resubmit their request form (e.g. email, admin alert, new status, etc)
- [ ] discuss options with the team
- [ ] create content or designs to accommodate the solution
- [ ] create dev ticket for implementation

### Additional Context

Sometimes analysts need to request additional information or an update within the request form. In that scenario, analysts change the request status to "Action needed." 

If the user sends the analyst additional information via email, the analyst will know when the request is ready to change back to "in review." However if the user made edits to the original request as part of the "action needed" (e.g. they updated the domain name in their request), then analysts need some type of notification that the action was completed. Currently, after a user resubmits the form (when in "action needed" status), the request returns to the "Submitted" status. Therefore, it could get lost in the shuffle.

**Comment by gabo0oo** [comment]
@h-m-f-t Would a ticketing system have any affect on this?

**Comment by h-m-f-t** [comment]
@gabo0oo yes, I expect it would influence the direction we take and how we instrument this. That remains an active line of investigation work. I could support closing this for now, or leaving it open and creating a new "ticketing system" label.

**Comment by gabo0oo** [comment]
@h-m-f-t I think it would be nice to leave it open but put it in the ticketing system bucket of work to be evaluated later.
